.devcontainer/pyproject.toml

```diff
@@ -5,15 +5,15 @@ description = "Epiphany Dev Container"
 authors = ["Epiphany Platform"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.10"
 pyyaml = "*"
 jinja2 = "*"
 boto3 = "*"
 jsonschema = "*"
 python-json-logger = "*"
-ansible = "==2.10.7"
 "ruamel.yaml" = "*"
 azure-cli = "^2.32.0"
+ansible = "^5.2.0"
```

Comment:
Maybe this may be be more specific, like using tilde instead of caret?

Reply:
Agreed, use '''poetry add ansible@5.2.0''' to fix it in place please. We only want to update this periodically, not every time we update the dependencies.

Reply:
Do we want to lock python to 3.10 as well? It was specified as ^3.7

Reply:
For me its ok, as this is coming from the container anyway which is fixed. What I did see is that I also did not lock the azure-cli version when I did the terraform update. Could you pls also remove and add azure-cli in again in this pull locking it to 2.32.0?

`poetry add azure-cli@2.32.0`

Reply:
This can be a good idea to lock this to python 3.10 as this may be additional safety net.

Reply:
So lets lock it.